### DISCUSSION TITLE
"MOT 2020 Labels" and "MOT 2020 Images"

### DISCUSSION THREAD

**Discussion by KalyanaVarma** [bdd100k/bdd100k#204] [Q&A]
Hi,

After extracting "MOT 2020 Labels" and "MOT 2020 Images" files,
I could find only 154 image folders with corresponding JSON files.
But there are many more JSON files without a corresponding folder with images.
How to get these folders of images.

Please let me know.

Thanks

**Comment by thomasehuang** [comment]
Can you make sure you clicked on MOT rather than MOTS? I just tested it and it should be fine. Let me know if you are still encountering this issue.
